packages/linker/src/handler.ts

```diff
@@ -1,12 +1,14 @@
 export type Domain = string | RegExp
 
+type LinkElements = HTMLAnchorElement | HTMLAreaElement
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg)
I think, it should not need `s`.  This type express **A** 'Element'

Reply:
i think so too